### DISCUSSION TITLE
Object detection list ?

### DISCUSSION THREAD

**Discussion by DemBones78** [ultralytics/yolov5#11784] [Q&A]
Hello,
I am trying to find a list of the object that (YOLOv5 6.2) can detect.
I am coming from this as a noob who knows nothing.

I have set up codeproject AI on my blueIris Server, it's working as intended.

However, I have a comma-separated text box where I can"list" objects.
So I have been searching for a list that's in the model to know what I can have it look for. 
but have not had any luck.    

am I looking at this or thinking about this wrong?

thanks for the help. 

![text](https://github.com/ultralytics/yolov5/assets/46800652/0294876f-5c7e-4131-91e8-876f49ba96d4)
![ai](https://github.com/ultralytics/yolov5/assets/46800652/4726a4b0-d7ce-4e1b-b04b-2031f0c175f8)

**Comment by ingus16** [comment]
person
bicycle, car, motorbike, aeroplane, bus, train, truck, boat
traffic light, fire hydrant, stop sign, parking meter, bench
cat, dog, horse, sheep, cow, elephant, bear, zebra, giraffe
backpack, umbrella, handbag, tie, suitcase, frisbee, skis, snowboard, sports ball, kite, baseball bat, baseball glove, skateboard, surfboard, tennis racket
bottle, wine glass, cup, fork, knife, spoon, bowl
banana, apple, sandwich, orange, broccoli, carrot, hot dog, pizza, donut, cake
chair, sofa, pottedplant, bed, diningtable, toilet, tvmonitor, laptop, mouse, remote, keyboard, cell phone, microwave, oven, toaster, sink, refrigerator, book, clock, vase, scissors, teddy bear, hair drier, toothbrush

Below are custom model object classes

IPcam-combined Labels: - person, bicycle, car, motorcycle, bus, truck, bird, cat, dog, horse, sheep, cow, bear, deer, rabbit, raccoon, fox, skunk, squirrel, pig

IPcam-general Labels (includes dark models images): - person, vehicle

IPcam-animal Labels: - bird, cat, dog, horse, sheep, cow, bear, deer, rabbit, raccoon, fox, skunk, squirrel, pig

IPcam-dark Labels: - Bicycle, Bus, Car, Cat, Dog, Motorcycle, Person
